output_template/python/run.py

```diff
@@ -133,34 +135,42 @@ def run_prediction(input_image, model, config_file, trial=1):
     pre_process = build_pre_process(config.PRE_PROCESSOR)
     post_process = build_post_process(config.POST_PROCESSOR)
 
-    # pre process for image
-    image_data, bench_pre = _timerfunc(_pre_process, (image_data, pre_process, config.DATA_FORMAT), trial)
+    results_total = []
+    results_pre = []
+    results_run = []
+    results_post = []
 
-    # add the batch dimension
-    image_data = np.expand_dims(image_data, axis=0)
+    funcs = [(_pre_process, image_data), _run, _post_process]
+
+    for i in range(trial):
+        # pre process for image
+        output, bench_pre = _timerfunc(_pre_process, (image_data, pre_process, config.DATA_FORMAT))
 
-    # run the model to inference
-    output, bench_inference = _timerfunc(_run, (nn, image_data), trial)
+        # run the model to inference
+        output, bench_run = _timerfunc(_run, (nn, output))
 
-    logger.info('Output: (before post process)\n{}'.format(output))
+        # pre process for output
+        output, bench_post = _timerfunc(_post_process, (output, post_process))
 
-    # pre process for output
-    output, bench_post = _timerfunc(_post_process, (output, post_process), trial)
+        results_total.append(bench_pre+bench_run+bench_post)
+        results_pre.append(bench_pre)
+        results_run.append(bench_run)
+        results_post.append(bench_post)
 
-    logger.info('Output: (after post process)\n{}'.format(output))
+    time_stat = {
+        "total": {"mean": np.mean(results_total), "std": np.std(results_total), },
+        "pre": {"mean": np.mean(results_pre), "std": np.std(results_pre), },
+        "post": {"mean": np.mean(results_post), "std": np.std(results_post), },
+        "run": {"mean":  np.mean(results_run), "std": np.std(results_run), },
+    },
```

Comment:
@tk26eng Sorry, I missed this.
This line looks return tuple. This is what you want?

```
In [1]: a = {"a": 0, "b": 1}
In [2]: b = {"a": 0, "b": 1},
In [3]: type(a)
Out[3]: dict
In [4]: type(b)
Out[4]: tuple
```

Reply:
Thank you.
Dict type is what I wanted to use.
I fixed this.

Reply:
@iizukak 
Can I go to Readability Review ?

Reply:
@tk26eng yes